config/valkyrie.yml

```diff
@@ -3,7 +3,7 @@ development:
   storage_adapter: disk
 
 test:
-  metadata_adapter: memory
+  metadata_adapter: postgres
```

Comment:
I think for our case it'd be best to just require having postgres up. We need it for Blacklight anyways, other projects require it, and it means we can make some assumptions about features (like transactions)